windows_containers/creating_windows_machinesets/creating-windows-machineset-nutanix.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="creating-windows-machineset-nutanix"]
+= Creating a Windows `MachineSet` object on Nutanix
+include::_attributes/common-attributes.adoc[]
+:context: creating-windows-machineset-nutanix
+
+toc::[]
+
+You can create a Windows `MachineSet` object to serve a specific purpose in your {product-title} cluster on Amazon Web Services (AWS). For example, you might create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines.
```

Comment:
You can create a different compute machine set to serve a specific purpose in your OpenShift Container Platform cluster on Nutanix